File: Java/src/test/java/Mock/MultipleDependencies.java
package Mock;

/**
 * Created by user on 21/9/2015.
 */
public class MultipleDependencies {
    public MockService1 service1 ;
    public MockService2 service2 ;

    public MultipleDependencies(MockService1 service1, MockService2 service2)
    {
        this.service1 = service1;
        this.service2 = service2;
    }
}

File: Container.Tests/ContainerTestClasses/MultipleDependencies.cs
using IoC.Tests.ContainerTestClasses.Mock;

namespace IoC.Tests.ContainerTestClasses
{
    class MultipleDependencies
    {
        public MockService1 Service1 { get; set; }
        public MockService2 Service2 { get; set; }

        public MultipleDependencies(MockService1 service1, MockService2 service2)
        {
            Service1 = service1;
            Service2 = service2;
        }
    }
}
